<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLembrete</class>
 <widget class="QWidget" name="FormLembrete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lembrete</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0,200);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widget_2{
border:1px solid blue;
	background-color: rgb(255, 255, 255);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="styleSheet">
            <string notr="true">QFrame#frame_6{
color:red;
border: none;
background-color:#2408D4;}

QLabel{color: white;
background-color:transparent;
font: 75 15pt &quot;MS Shell Dlg 2&quot;;
border:none
}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>INCLUIR LEMBRETE</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>80</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(220, 220, 220);
border:none;
color:black</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QToolButton" name="amarela">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="text">
               <string>amarelo</string>
              </property>
              <property name="icon">
               <iconset resource="../assets/file.qrc">
                <normaloff>:/pagina/telainicial/paginaamarela.png</normaloff>:/pagina/telainicial/paginaamarela.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="azul">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="text">
               <string>azul</string>
              </property>
              <property name="icon">
               <iconset resource="../assets/file.qrc">
                <normaloff>:/pagina/telainicial/paginaazul.png</normaloff>:/pagina/telainicial/paginaazul.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="laranja">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="text">
               <string>laranja</string>
              </property>
              <property name="icon">
               <iconset resource="../assets/file.qrc">
                <normaloff>:/pagina/telainicial/paginalaranja.png</normaloff>:/pagina/telainicial/paginalaranja.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="preto">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="text">
               <string>preto</string>
              </property>
              <property name="icon">
               <iconset resource="../assets/file.qrc">
                <normaloff>:/pagina/telainicial/paginapreta.png</normaloff>:/pagina/telainicial/paginapreta.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="verde">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="text">
               <string>verde</string>
              </property>
              <property name="icon">
               <iconset resource="../assets/file.qrc">
                <normaloff>:/pagina/telainicial/paginaverde.png</normaloff>:/pagina/telainicial/paginaverde.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="vermelho">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="text">
               <string>vermelho</string>
              </property>
              <property name="icon">
               <iconset resource="../assets/file.qrc">
                <normaloff>:/pagina/telainicial/paginavermelha.png</normaloff>:/pagina/telainicial/paginavermelha.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_corest">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(207, 207, 207);
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(218, 218, 218);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
	background-color: rgb(223, 223, 223);
}
QPushButton{
background-color:rgba(2, 65, 118, 255);
color:rgba(255, 255, 255, 200);
border-radius:5px;
padding:15px;
	font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2, 65, 118, 100);
background-position:calc(100% - 10px)center;
}
QPushButton:hover{
background-color:rgba(2, 65, 118, 200);
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="salva">
              <property name="text">
               <string>SALVAR (F2)</string>
              </property>
              <property name="shortcut">
               <string>F2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bt_altera">
              <property name="text">
               <string>ALTERAR (F2)</string>
              </property>
              <property name="shortcut">
               <string>F2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cancelar">
              <property name="text">
               <string>CANCELAR (ESC)</string>
              </property>
              <property name="shortcut">
               <string>Esc</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../assets/file.qrc"/>
 </resources>
 <connections/>
</ui>
